settings/PandoraSettings_Inference_DUNEFD_DeepLearningId.xml

```diff
@@ -0,0 +1,32 @@
+<pandora>
```

Comment:
Will need to think of a longer-term strategy about how to deploy things with and without DL. This could go in settings/development right now, or even a settings/dl or similar.

In the longer term, we'll likely want to be offering (and this will be DUNE FD neutrino pass at first, I'd expect) a complete reco option _with_ and a complete reco option _without_ DLContent. One day we'd expect to maintain the non-DL version just as a legacy option.

Reply:
Yes, at the moment this is just a minimal example of running the network, so it can be moved elsewhere.